lib/ohai/plugins/vmware.rb

```diff
@@ -53,6 +53,17 @@ def get_vm_attributes(vmtools_path)
         # to attribute "vmware[:<parameter>]"
         %w{hosttime speed sessionid balloon swap memlimit memres cpures cpulimit}.each do |param|
           vmware[param] = from_cmd([vmtools_path, "stat", param])
+          if param == 'hosttime' && vmtools_path =~ /Program Files/
+            # popen and %x return stdout encoded as IBM437 in Windows but in a string marked
+            # UTF-8. The string doesn't throw an exception when encoded to "UTF-8" but
+            # displays [?] character in Windows without this.
+            #
+            # .force_encoding(Encoding::ISO_8859_1) causes the character to be dropped
+            # and .force_encoding(Encoding::Windows_1252) displays the „ character in place
+            # of an ä. .force_encoding(Encoding::IBM437) allows for the correct characters
```

Comment:
- Note that this is broken through Windows Server 2022.
- Search and/or open bug with ruby-core on `popen` and `Time.zone`